src/nvme/tree.c

```diff
@@ -202,8 +200,7 @@ int nvme_host_get_ids(nvme_root_t r,
 
 nvme_host_t nvme_default_host(nvme_root_t r)
 {
-	_cleanup_free_ char *hostnqn = NULL;
-	_cleanup_free_ char *hostid = NULL;
+	_cleanup_free_ char *hostnqn = NULL, *hostid = NULL;
```

Comment:
another one of code churn, nothign gained, makes the review more harder, makes backporting harder.